src/input/mpegts/iptv/iptv_auto.c

```diff
@@ -103,6 +103,10 @@ iptv_auto_network_process_m3u_item(iptv_network_t *in,
 
   epgid = htsmsg_get_str(item, "tvh-chnum");
   chnum2 = epgid ? prop_intsplit_from_str(epgid, CHANNEL_SPLIT) : 0;
+
+  htsmsg_get_s32(item, "tvh-prio", &muxprio);
```

Comment:
Use htmsg_get_s32_or_default() otherwise the variable has an unknown value when tvh-prio is not defined.